{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2023-46570"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "An out-of-bounds read in radare2 v.5.8.9 and before exists in the print_insn32 function of libr/arch/p/nds32/nds32-dis.h."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://gist.github.com/gandalf4a/d7fa58f1b3418ef08ad244acccc10ba6",
          "refsource": "MISC",
          "tags": [],
          "url": "https://gist.github.com/gandalf4a/d7fa58f1b3418ef08ad244acccc10ba6"
        },
        {
          "name": "https://github.com/radareorg/radare2/issues/22333",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/radareorg/radare2/issues/22333"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2023-10-29T01:44Z",
  "publishedDate": "2023-10-28T02:15Z"
}